cgroup/cgroup.go

```diff
@@ -0,0 +1,77 @@
+package cgroup
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/godbus/dbus/v5"
+	"github.com/godbus/dbus/v5/introspect"
+	"github.com/godbus/dbus/v5/prop"
+
+	logging "github.com/home-assistant/os-agent/utils/log"
+)
+
+const (
+	objectPath            = "/io/homeassistant/os/CGroup"
+	ifaceName             = "io.homeassistant.os.CGroup"
+	CGroupFSDockerDevices = "/sys/fs/cgroup/devices/docker/"
+)
+
+type cgroup struct {
+	conn *dbus.Conn
+}
+
+func (d cgroup) UpdateDevicesAllowed(containerID string, permission string) (bool, *dbus.Error) {
```

Comment:
The file `devices.allow` only allows to add devices allowed, not remove. Not sure if that is ok for our use case...

If we keep the functionality as is, I would rename the function. `UpdateDevicesAllowed` sounds like one can update the list of allowed devices, where in fact only allowed deviecs can be added. How about

```suggestion
func (d cgroup) AddDevicesAllowed(containerID string, permission string) (bool, *dbus.Error) {
```